pyproject.toml

```
@@ -47,6 +47,7 @@ dependencies = [
   "sqlalchemy>=1.2.0,<1.3.0",
   "textacy~=0.10.0",  # no higher until most_discriminating_terms replacement secured
   "webargs~=1.5",
+  "pymupdf>=1.22.3"
```

i've been trying to constrain our dependencies a bit more tightly, to avoid the mess we're in now with the production environment. :) considering [this package's changelog](https://pymupdf.readthedocs.io/en/latest/changes.html), could you change this to 

```suggestion
  "pymupdf~=1.22.0"
```